regression/contracts/assigns_enforce_18/test.desc

```diff
@@ -0,0 +1,22 @@
+CORE
+main.c
+--enforce-all-contracts _ --pointer-primitive-check
+^EXIT=10$
+^SIGNAL=0$
+^\[bar.\d+\] line \d+ Check that \*b is assignable: SUCCESS$
```

Comment:
We don't check that `a` is assignable at `free(a)`, or may be we do but don't have a regex for it?

Reply:
Good catch!! I just sent a new commit to check frame conditions on free function calls.